src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -97,9 +97,15 @@ public function process(ContainerBuilder $container)
 
             // We must assume that the class value has been correctly filled, even if the service is created by a factory
             $class = $container->getParameterBag()->resolveValue($def->getClass());
-
             $interface = 'Symfony\Component\EventDispatcher\EventSubscriberInterface';
+
             if (!is_subclass_of($class, $interface)) {
+                if (!class_exists($class, true)) {
+                    throw new \RuntimeException(
+                        sprintf('Class "%s" from definition of service "%s" can not be found. Check existence of file and correct naming', $class, $id)
```

Comment:
I mean: when a class fails to be loaded, regular PHP bails out with a short "class not found". I don't see much value in adding some suggestions to fix the issue, especially since the reason might be completely different and we could hint in the wrong direction. Let's stick to what we now for sure: the class is missing. But we don't know why in this very code path.